src/routes/auth/index.js

```diff
@@ -3,20 +3,15 @@ import local from './local';
 import facebook from './facebook';
 import twitter from './twitter';
 import google from './google';
-import status from '../../config/status';
+import logout from '../../middlewares/logout';
+import verifyToken from '../../middlewares/verifyToken';
 
 const router = express.Router();
 
 router.use('/', local);
 router.use('/facebook', facebook);
 router.use('/twitter', twitter);
 router.use('/google', google);
-
-router.get('/logout', (req, res) => {
-  req.logout();
-  res.status(status.OK).json({
-    message: 'you are successfuly logged out'
-  });
-});
+router.get('/logout', verifyToken, logout);
```

Comment:
What if you provide routes to manually sign out an account. 
For instance, someone can be logged in using different devices, and they may want to sign out one of the devices he wants. and also, admins can be able to force-signout an account if they want. What do you think?